docs/ops/security-ssl.zh.md

```diff
@@ -129,7 +129,7 @@ security.ssl.internal.truststore: /path/to/file.truststore
 security.ssl.internal.truststore-password: truststore_password
 {% endhighlight %}
 
-When using a certificate that is not self-signed, but signed by a CA, you need to use certificate pinning to allow only a
+When using a certificate that is not self-signed, but signed by a CA, you need to use certificate pinning to allow only a 
```

Comment:
what changed here?

Reply:
Nothing, this is an artifact of how I did the changes: I changed the English version, then copied over to the Chinese version. I left the whitespace change in because it will keep re-occuring in the future if we again copy changes over.

I can also remove it, of course, though.